src/vscode-bicep/src/test/e2e/commands.ts

```diff
@@ -55,3 +55,9 @@ export async function executeBuildCommand(
 ): Promise<void> {
   return await vscode.commands.executeCommand("bicep.build", documentUri);
 }
+
+export async function executeDeployCommand(
```

Comment:
Not used?  We probably we need e2e tests for deploy (mocked) given the amount of code, do you agree?  That requires integrating some more code from AzureTools repo, 'dev' module.  TestUserInput I believe.

Reply:
Yup, totally agree. As discussed, offline, will add some tests in a follow up PR. Created below issue to track the work:
https://github.com/Azure/bicep/issues/6217

Reply:
Removed the unused command here:
https://github.com/Azure/bicep/commit/966be896f4b948a0ae2cfc40daa9b8bb66cae1ce